code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -1692,19 +1749,91 @@ func (srh *StorageRestHandler) getBlobbers(w http.ResponseWriter, r *http.Reques
 	common.Respond(w, r, sns, nil)
 }
 
-// todo add filter or similar
-// swagger:route GET /v1/screst/6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7/get_blobber_lat_long get_blobber_lat_long
-// Gets list of latitude and longitude for all blobbers
+// swagger:route GET /v1/screst/6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7/blobbers-by-geolocation blobbers-by-geolocation
+//
+//  Returns a list of all blobbers within a rectangle defined by maximum and minimum latitude and longitude values.
+//
+//    + name: max_latitude
+//      description: maximum latitude value, defaults to 90
+//      in: query
+//      type: string
+//    + name: min_latitude
+//      description:  minimum latitude value, defaults to -90
+//      in: query
+//      type: string
+//    + name: max_longitude
+//      description: maximum max_longitude value, defaults to 180
+//      in: query
+//      type: string
+//    + name: min_longitude
+//      description: minimum max_longitude value, defaults to -180
+//      in: query
+//      type: string
 //
 // responses:
-//  200: BlobberLatLong
+//  200: stringArray
 //  500:
-func (srh *StorageRestHandler) getBlobberGeoLocation(w http.ResponseWriter, r *http.Request) {
+func (srh *StorageRestHandler) getBlobbersByGeoLocation(w http.ResponseWriter, r *http.Request) {
+	var maxLatitude, minLatitude, maxLongitude, minLongitude float64
+	var err error
+
+	maxLatitudeString := r.URL.Query().Get("max_latitude")
+	if len(maxLatitudeString) > 0 {
+		maxLatitude, err = strconv.ParseFloat(maxLatitudeString, 64)
+		if err != nil {
+			common.Respond(w, r, nil, common.NewErrBadRequest("bad max latitude"+err.Error()))
+		}
+		if maxLatitude > MaxLatitude {
+			common.Respond(w, r, nil, common.NewErrBadRequest("max latitude "+maxLatitudeString+" out of range -90,+90"))
+		}
+	} else {
+		maxLatitude = MaxLatitude
+	}
+
+	minLatitudeString := r.URL.Query().Get("min_latitude")
+	if len(minLatitudeString) > 0 {
+		minLatitude, err = strconv.ParseFloat(minLatitudeString, 64)
+		if err != nil {
+			common.Respond(w, r, nil, common.NewErrBadRequest("bad max latitude"+err.Error()))
+		}
+		if minLatitude < MinLatitude {
+			common.Respond(w, r, nil, common.NewErrBadRequest("max latitude "+minLatitudeString+" out of range -90,+90"))
+		}
+	} else {
+		minLatitude = MinLatitude
+	}
+
+	maxLongitudeString := r.URL.Query().Get("max_longitude")
+	if len(maxLongitudeString) > 0 {
+		maxLongitude, err = strconv.ParseFloat(maxLongitudeString, 64)
+		if err != nil {
+			common.Respond(w, r, nil, common.NewErrBadRequest("bad max longitude"+err.Error()))
```

Comment:
As above.

There should be a space between the `longitude"+err.Error()`, i.e., `longitude " + err.Error()`